### PR TITLE
Add Getting started guide

### PR DESCRIPTION
**Pull request by javiber**
- Changed a parameter of material mkdocs to make top-level navigation tabs. IMO it's a much better experience than nested navigation on the left. We don't have a preview build based on a PR yet but I will add a screenshot.
- Added a getting started section that contains a lot of info, probably a lot of things to improve in that section. 
- Changed the `/reference` page which was not navigatable before to contain the docstring from norfair. Open to suggestions for better alternatives though.
Selection_061

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+To debug this use [`draw_points`][norfair.drawing.draw_points] or [`draw_boxes`][norfair.drawing.draw_boxes] to inspect the detections and analyze if they are precise enough. If you are filtering the detections based on scores, this is a good time to tweak the threshold. If you decide that the detections are not good enough you can try a different architecture, a bigger version of the model, or consider fine-tuning the model on your domain.
+
+
+#### Tracking Issues
+
+After inspecting the detections you might find issues with the tracking, several things can go wrong with tracking but here is a list of common errors and things to try:
+
+- Objects take **too long to start**, this can have multiple causes:
+    - `initialization_delay` is too big on the Tracker. Makes the TrackedObject stay on initializing for too long, `3` is usually a good value to start with.
+    - `distance_threshold` is too big on the Tracker. Prevents the Detections to be matched with the correct TrackedObject. The best value depends on the distance used.
```
Comment (facundo-lezama) [inline docs/getting_started.md:78]: Didn't get the idea behind this recommendation. Shouldn't it be that the `distance_threshold` is too **small**?

**Review by facundo-lezama — COMMENTED** [review]
It looks great @javiber! Thanks for adding the Getting Started section, it is an excellent addition to Norfair's documentation. 

I left a bunch of comments, mostly formatting issues, but apart from that, I think it's ready to merge. I assumed the formatting was expected to work as in Markdown, I didn't try it in the actual documentation, so feel free to dismiss the comments if you checked that the formatting was correct.

File: docs/getting_started.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+- [Alphapose](https://github.com/tryolabs/norfair/tree/master/demos/alphapose)
+- [Openpose](https://github.com/tryolabs/norfair/tree/master/demos/openpose)
+- [MMDetection](https://github.com/tryolabs/norfair/tree/master/demos/mmdetection)
+
+Any other model trained on one of the supported tasks is also supported and should be easy to integrate with Norfair, regardless of whether it uses Pytorch, TensorFlow, or other.
+
+If you are unsure of which model to use, [Yolov7](https://github.com/WongKinYiu/yolov7) is a good starting point since it's easy to set up and offers models of different sizes pre-trained on object detection and pose estimation.
+
+!!! Note
+    Norfair is a Detection-Based-Tracker (DBT) and as such, its performance is highly dependent on the performance of the model of choice.
```
[the remark this answers is not in the document]
Comment (javiber) [reply docs/getting_started.md:21]: The `!!! Note` is not standard markdown and I don't think Github renders it correctly but it's part of our [mkdocs theme](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#usage)

File: docs/getting_started.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+The detections from the model will need to be wrapped in an instance of [Detection][norfair.tracker.Detection] before passing them to Norfair.
```
[the remark this answers is not in the document]
Comment (javiber) [reply docs/getting_started.md:23]: This is how to define links to other sections in [mkdocstring](https://mkdocstrings.github.io/usage/#cross-references)

**Review by facundo-lezama — APPROVED** [review]

**Pull request merged**